incubator/prometheus/templates/NOTES.txt

```diff
@@ -0,0 +1,74 @@
+The Prometheus server can be accessed via port {{ .Values.server.httpPort }} on the following DNS name from within your cluster:
+{{ template "server.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
+{{ if .Values.server.ingress.enabled -}}
+From outside the cluster, the server URL(s) are:
+{{- range .Values.server.ingress.hosts }}
+http://{{ . }}
+{{- end }}
+
+{{- else }}
+Get the server URL by running these commands in the same shell:
+{{- if contains "NodePort" .Values.server.serviceType }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "server.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+
+{{- else if contains "LoadBalancer" .Values.server.serviceType }}
+**** NOTE: It may take a few minutes for the LoadBalancer IP to be available.                             ****
```

Comment:
Can you please remove the stars from NOTES.txt they end up getting rendered in a strange way.